### PR TITLE
feat: add Mode network and refactor chain data

### PR DESCRIPTION
**Pull request by 0xArdy**
closes SUP-96

### CODE REVIEW

File: src/utils/client.ts [hunk 1]
```diff
@@ -3,36 +3,16 @@ import { connectorsForWallets } from '@rainbow-me/rainbowkit';
 import * as wagmiChains from 'wagmi/chains';
 import { createConfig, cookieStorage, createStorage } from 'wagmi';
 import { Transport, http } from 'viem';
-import { sepolia, mainnet } from 'viem/chains';
-import { optimismSepolia, baseSepolia, optimism, base, zora, fraxtal } from 'viem/op-stack';
+import { sepolia } from 'viem/chains';
```
Comment (0xtiti) [inline src/utils/client.ts:6]: why only sepolia?

**Review by 0xtiti — APPROVED** [review]

File: src/utils/client.ts (hunk 1 above)
Comment (0xArdy) [reply src/utils/client.ts]: removed, its no longer necessary https://github.com/defi-wonderland/superchain-bridge-ui/pull/59/commits/41a9a9bafa564fc75633b391de0ad4e6ea3ed4f7

**Review by 0xtiti — APPROVED** [review]

File: src/data/extraTokens.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  {
+    chainId: 11155111,
+    address: '0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238',
+    name: 'Native USDC',
+    symbol: 'USDC',
+    decimals: 6,
+    logoURI: 'https://assets.coingecko.com/coins/images/6319/standard/usdc.png',
+    cctp: true,
+  },
```
Comment (turtlemoji) [inline src/data/extraTokens.ts:30]: just remionder to recheck

File: src/data/chains/zora.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    portal: zora.contracts.portal[mainnet.id].address,
+  },
+
+  // ------------------ L2 to L1 ------------------
+  // zora-->ethereum
+  [`${zora.id}-${mainnet.id}`]: l2Contracts,
+
+  // zoraSepolia-->sepolia
+  [`${zoraSepolia.id}-${sepolia.id}`]: l2Contracts,
+};
```
Comment (turtlemoji) [inline src/data/chains/zora.ts:40]: noticed a pattern with other chains, maybe it can be abstracted on another PR to a helper function later

**Review by turtlemoji — APPROVED** [review]
approved! double check addresses tho @0xArdy

**Pull request merged**